homeassistant/components/sabnzbd/config_flow.py

```diff
@@ -36,7 +36,7 @@
 class SABnzbdConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     """Sabnzbd config flow."""
 
-    VERSION = 1
+    VERSION = CONFIG_ENTRY_VERSION
```

Comment:
There is nothing changing in the config flow so you should remain on version 1.
Also no point in using a constant for this.
```suggestion
    VERSION = 1
```

Reply:
But if I don't bump the version the migration flow won't trigger

Reply:
Then I would suggest you move the relevant section to `setup_entry` instead as you don't need a config migration.

Reply:
So the migrate method is only for config flow? But if I move the code to setup entry wouldn't it trigger on each HA start?

Reply:
It would so you need some guard in place.

Reply:
Done